.github/workflows/ci-backend.yml

```diff
@@ -0,0 +1,35 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI_BACKEND
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the development branch
+  push:
+    branches: [ development ]
+  pull_request:
+    branches: [ development ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
```

Comment:
is the value supposed to be blank or should it say false or something?

Reply:
I think usually there would be options to add under worflow_dispatch, but for now, I think we just need to add the manual trigger.
https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/

Reply:
having a tag for each manual run would be helpful to know what the purpose was. So maybe we can add a key like this to the workflow_dispatch
```
tags:
        description: 'Test scenario tags' 
```

Reply:
Sounds good, I'll add that.